packages/instantsearch.css/package.json

```diff
@@ -21,12 +21,22 @@
     "styles:autoprefixer": "postcss themes/ --use autoprefixer --replace --no-map",
     "styles:minify": "sass src/themes/algolia.scss:themes/algolia-min.css src/themes/satellite.scss:themes/satellite-min.css src/themes/reset.scss:themes/reset-min.css --no-source-map --style compressed",
     "styles:build": "sass src/themes:themes --no-source-map",
-    "prepare": "yarn build"
+    "prepare": "yarn build",
+    "lint": "stylelint \"**/*.scss\""
```

Comment:
When we have taken care of all the blocking violations, we can add `&& lerna run lint` to the `lint` command of our main `package.json`.